projects/demo/src/modules/directives/dropdown/dropdown.template.html

```diff
@@ -8,7 +8,7 @@
 
         <p i18n>
             See also
-            <a tuiLink [routerLink]="['/directives/hosted-dropdown']"
+            <a tuiLink [routerLink]="['/components/hosted-dropdown']"
```

Comment:
@nsbarsukov e2e tests don't cover this case?

Reply:
They don't go over links on pages :)

Reply:
@splincode where exactly do you want me to update the test?

Reply:
@hakimio can you write a simple e2e test
@waterplea if needed? ))

Reply:
Yes, I can. Should it be in `demo-integrations/cypress/tests/demo/dropdown` or somewhere else?
@waterplea do you want to have the test?

Reply:
I don't think this is necessary

Reply:
> @nsbarsukov e2e tests don't cover this case?

I think that covering the `demo` app with tests is redundant. The main thing for us is to cover our components (which we publish as the library) with tests.